build.sh

```diff
@@ -2,7 +2,7 @@
 cd "$(dirname "$0")"
 
 # Use all cores for the build process
-CORE_COUNT=$(cat /proc/cpuinfo | grep processor | wc -l)
```

Comment:
@sptndc I'm failing to see any output when running `< /proc/cpuinfo | grep -c processor -l`. Same with `echo $(< /proc/cpuinfo | grep -c processor -l)`.

Secondly, `-l` in grep is for "Display the filenames, but do not display the matched lines." -- is that the intended use-case or a left-over from the `wc -l`?

Reply:
sorry i forgot to remove `-l` from there.

Reply:
`echo $(< /proc/cpuinfo | grep -c processor)` still returns `0` on a Debian Wheezy box so I'll revert it back to `cat` for now.

Reply:
Ok, no problem. It's just an issue report from `shellcheck`.

Reply:
That's a great idea -- we should probably make Travis run the `spellcheck` on pull requests.

Reply:
Yes, that's better.